source/tls_channel_handler.c

```diff
@@ -272,8 +264,27 @@ int aws_tls_ctx_options_init_client_mtls_with_pkcs11(
         aws_string_destroy(pkcs_private_key_object_label);
     }
 
+    struct aws_byte_buf tmp_cert_buf;
+    if (aws_byte_buf_init(&tmp_cert_buf, allocator, 0) != AWS_OP_SUCCESS) {
```

Comment:
this is one of those functions that can't fail, so don't need error checking

actually you don't need this function call at all, you double-initialized this byte-buf

all you need is the aws_byte_buf_init_copy_from_cursor() call